server/repositories/browserscope.js

```diff
@@ -22,10 +22,14 @@ exports.register = function (server, options, next) {
           try {
             resolve(JSON.parse(str).test_key);
           } catch (e) {
-            reject(new Error('Unexpected response from browserscope.org'));
+            server.log('error', e);
```

Comment:
I like to avoid single-letter variable names, like `exception` instead of `e`. WDYT about that?